### PR TITLE
Global footer

### PR DESCRIPTION
**Pull request by softsimon**
The footer is now global at the bottom on all pages, except the About page.

### CODE REVIEW

File: frontend/src/app/components/master-page/master-page.component.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export class MasterPageComponent implements OnInit {
     this.urlLanguage = this.languageService.getLanguageForUrl();
     this.subdomain = this.enterpriseService.getSubdomain();
     this.navigationService.subnetPaths.subscribe((paths) => {
+      console.log(paths);
```
Comment (bufo24) [inline frontend/src/app/components/master-page/master-page.component.ts:38]: Remove logging
```suggestion
```

**Review by hunicus — COMMENTED** [review]
This will be very helpful to have for #3446.

Some findings...

Footer can appear to the side of the sidebar for docs on desktop when window:

Screenshot from 2023-03-31 01-30-43

Footer is not on about page:

Screenshot from 2023-03-31 01-31-10

Footer clashes with bottom of docs page on mobile:

Screenshot from 2023-03-31 02-44-36

**Review by wiz — CHANGES_REQUESTED** [review]
Missing footer on liquid pages

**Comment by hunicus** [comment]
It looks like the footer isn't working on the Liquid site because of an unrelated error in `asset-circulation.component.ts` that occurs before the changes in this pull request.

How can we proceed?

**Review by hunicus — APPROVED** [review]
ACK @ https://github.com/mempool/mempool/pull/3564/commits/4c5ece82491b1df880e591fe0479338e754cf9fb

**Pull request merged**